students/uss_tomasz/lesson_11_argparse/11_1_regex_search/test_regex_search.py

```diff
@@ -0,0 +1,71 @@
+import os
+import shutil
+import tempfile
+import unittest
+from regex_search import parse_args, get_file_names_list, find_regex_in_txt_files, execute_script
+
+
+class TestRegexSearchNoFiles(unittest.TestCase):
+    def setUp(self):
+        self.test_dir = tempfile.mkdtemp()
```

Comment:
You can also change working directory to this temporary directory in order to test if everything works fine with default params
![image](https://user-images.githubusercontent.com/57070/71582481-b6841c00-2b0a-11ea-9984-d9d99fa8fa09.png)